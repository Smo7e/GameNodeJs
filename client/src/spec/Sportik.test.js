import React from "react";
import { render } from "@testing-library/react";
import Sportik from "../component/Heroes/component/Sportik"; 

describe("Sportik Component", () => {
  it("renders correctly", () => {
    const Heroes = jest.fn(); 
    const { asFragment } = render(<Sportik heroes={Heroes} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
