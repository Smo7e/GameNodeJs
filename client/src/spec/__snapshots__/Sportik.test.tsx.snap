// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sportik Component renders correctly 1`] = `
<DocumentFragment>
  <div>
    <button
      class="arrow-2-heroes"
    />
    <div
      class="image-Sportik-heroes"
    />
    <div
      class="panel-heroes"
    >
      <div
        class="text-heroes"
      >
        “Спортик” - имеет повышенное здоровье в 150 единиц, увеличенный вдвое урон по паре “Физра” и любое улучшение от предметов увеличивается в 1.25 раза.
      </div>
    </div>
  </div>
</DocumentFragment>
`;
